### PR TITLE
Guessing number and print day in word

### PR DESCRIPTION
**Pull request by SrijanBajracharya**

### CODE REVIEW

File: src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package com.lftechnology.java.training.srijan.basics;
+
+
+import java.security.SecureRandom;
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java:4]: You can also use Math.Random to generate numbers between [0,1]. You can then change the logic to generate numbers between any interval.

Class SecureRandom is used to generate random numbers for security purposes and it is more time consuming. For purposes like these, i would recommend Math.Random

File: src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.Scanner;
+import java.util.logging.Logger;
+/**
+ * Write a Guessing a number game application: 
+ * application will generate a random no between 1 and 20. 
+ * user guess the value by entering from the console
+ * Check if the guess value matched with random number or not, 
+ * if matched show the guessing value, random value and the no. of attempts.
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java:12]: Add author

File: src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	}
+
+	public static void main(String[] args) {
+		PrintDayInWord daysInWeek = new PrintDayInWord();
+		int dayInNumber;
+
+		Scanner input = null;
+
+		
+			input = new Scanner(System.in);
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java:35]: Why are you initializing Scanner in different lines if they are in same block? We do this when the initialization needs to be done in different block viz  try block.

File: src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	private static final Logger LOGGER = Logger.getLogger(GuessingNumbers.class.getName());
+	
+	public GuessingNumbers(){
+		numberOfAttempt = 0;
+	}
+	
+	public static void main(String[] args){
+		Scanner input = null;
+		SecureRandom randomNumber = new SecureRandom();
+		GuessingNumbers guessNumber = new GuessingNumbers();
```
Comment (Msanjib) [inline src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java]: why are you making object  of GuessingNumbers inside GuessingNumber method :question: 
The main method should be in different class.

File: src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+		int guessedNumber;
```
Comment (bibhushanJoshi) [inline src/com/lftechnology/java/training/srijan/basics/GuessingNumbers.java:26]: initialization is required.
ie guessedNumber = 0;

File: src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+		daysInWeek[2] = "Tuesday";
+		daysInWeek[3] = "Wednesday";
+		daysInWeek[4] = "Thursday";
+		daysInWeek[5] = "Friday";
+		daysInWeek[6] = "Saturday";
+
+	}
+
+	public static void main(String[] args) {
+		PrintDayInWord daysInWeek = new PrintDayInWord();
```
Comment (bibhushanJoshi) [inline src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java:29]: why is PrintDayInWord object being created inside PrintDayInWord class ??
This method should be in separate class.

File: src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+		
+			input = new Scanner(System.in);
+			
+			LOGGER.info("Please Enter a Number to represent day in a week ");
+			dayInNumber = input.nextInt();
+
+			switch (dayInNumber) {
+
+			case 0: {
+				LOGGER.info("0 corresponds to" + " " + daysInWeek.daysInWeek[0]);
```
Comment (bibhushanJoshi) [inline src/com/lftechnology/java/training/srijan/basics/PrintDayInWord.java:43]: better to use parameterized logger
http://stackoverflow.com/questions/17716381/slf4j-parameterized-logging-using-varargs-method

**Pull request merged**